packages/trader/src/App/Components/Elements/VideoStream/volume-control.tsx

```diff
@@ -0,0 +1,177 @@
+import React from 'react';
+import { CSSTransition } from 'react-transition-group';
+import { Icon } from '@deriv/components';
+
+type TVolumeControl = {
+    onVolumeChange: (new_value: number) => void;
+    volume?: number;
+    is_mobile?: boolean;
+    toggleMute: (new_value: boolean) => void;
+};
+
+const VolumeControl = ({ onVolumeChange, volume, is_mobile, toggleMute }: TVolumeControl) => {
+    const [show_volume, setShowVolume] = React.useState(false);
+    const [is_muted, setIsMuted] = React.useState(false);
```

Comment:
Let's place this useState in the parent `VideoPlayer` component so that we don't need extra callback `toggleMute`. We can make it controlled using only `is_muted` and `setIsMuted` :)